Riot/Assets/en.lproj/Vector.strings

```diff
@@ -84,10 +84,10 @@
 "auth_invalid_email" = "This doesn't look like a valid email address";
 "auth_invalid_phone" = "This doesn't look like a valid phone number";
 "auth_missing_password" = "Missing password";
-"auth_add_email_message" = "Add an email address to your account to let users discover you, and to reset your password.";
-"auth_add_phone_message" = "Add a phone number to your account to let users discover you.";
-"auth_add_email_phone_message" = "Add an email address and/or a phone number to your account to let users discover you. Email address will also let you reset your password.";
-"auth_add_email_and_phone_message" = "Add an email address and a phone number to your account to let users discover you. Email address will also let you reset your password.";
+"auth_add_email_message_2" = "Set an email for account recovery. Use email to optionally be discoverable by existing contacts.";
```

Comment:
I really find these two sentences unclear.
What about something like: "Add an email to your account. We will be able to connect to your account with this email. And you can later in the settings make this email publicly visible to be discoverable in the Matrix ecosystem by other users who know your email."

Reply:
Let's just talk about account recovery then. "be discoverable by existing contacts" can't be less clear to me. Which contacts are we talking about?

Reply:
I think it's better (it's not people you know prior to using Riot, but people who know you. It's not the same thing :))

Reply:
I still prefer my suggestion, because it explains the advantage to add an email, in a quite empty screen (so we have room for a long wording) and add a reminder to make it public in the settings. It just misses the recovery aspect. The `later` suggest that it's not possible yet

Reply:
Yes, it's clearer (for me at least) but for the end of the sentence: `...by people who know you` ?

Reply:
or `by others`

Reply:
Yes, that's why I proposed `users who know your email.` which is precise :)